### DISCUSSION TITLE
数学行间公式渲染不了呜呜

### DISCUSSION THREAD

**Discussion by Dxin02** [YunYouJun/hexo-theme-starry#316] [Q&A 疑难杂症]
![image](https://user-images.githubusercontent.com/87813631/172803938-dadc0a61-82db-4ecb-99ba-db5916aff296.png)

**Comment by Dxin02** [comment]
在Typora里可以正常显示。
![image](https://user-images.githubusercontent.com/87813631/172804983-7e821448-40e3-4e4e-8073-4696216be799.png)

**Reply by YunYouJun** [reply]
尝试使用 `<div></div>` 包裹。

```latex
<div>
$$
\begin{equation}
  \left\{
    \begin{aligned}
      x=a\cos\theta\\
      y=b\sin\theta\\
    end{aligned}
  \right.
\end{equation}
$$
</div>
```
